samples/msgext-ai-sentiment-analysis/nodejs/infra/azure.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
```

Comment:
Not sure if we need this.

Reply:
This folder will generate for bot samples by default, so we pushing this files and i referred yours PR as well.